src/utils/Processing.ts

```diff
@@ -168,6 +170,90 @@ export function writeFHIRResources(outDir: string, outPackage: Package, snapshot
   logger.info(`Exported ${count} FHIR resources as JSON.`);
 }
 
+/**
+ * Initializes an empty sample FSH within a user specified subdirectory of the current working directory
+ */
+export function init(): void {
+  console.log(
+    '\n╭──────────────────────────────────────────────────────────╮\n' +
+      '│ This interactive tool will use your answers to create a  │\n' +
+      "│ working SUSHI project configured with your project's     │\n" +
+      '│ basic information. See the SUSHI documentation for more  │\n' +
+      '│ guidance on project structure and configuration.         │\n' +
+      '╰──────────────────────────────────────────────────────────╯\n'
```

Comment:
I hate it when I find issues and realize that you were just doing _exactly_ as I told you.  Doh!

Well, anyway... I _meant_ to delete the last sentence in this box since I put a similar thing in the box when the process completes.  I don't think we need to tell them twice -- and I think it's more appropriate in the second box.  So I'd strike the last sentence from this box.